clusterctl/examples/google/generate-yaml.sh

```diff
@@ -119,6 +119,12 @@ gcloud projects add-iam-policy-binding $GCLOUD_PROJECT --member=serviceAccount:$
 echo Generating $WORKER_SA_EMAIL service account for workers
 gcloud iam service-accounts create --display-name="worker service account" $WORKER_SA_NAME
 
+FIREWALL_RULE_NAME="cluster-api-open"
+if ! gcloud compute firewall-rules describe ${FIREWALL_RULE_NAME}; then
```

Comment:
(nit)
All the other conditions seem to be using brackets. Can we use it here too for consistency?